expressions/variables.md

```diff
@@ -57,6 +57,14 @@ x = 5  // OK
 y = 6  // Error, y is let
 ```
 
+Using `let`instead of `var` also means the variable has to be assigned immediatly. 
```

Comment:
A few typos here:
- need a space after `let`
- "immediatly" is spelled incorrectly